tests/dags/providers/provider_api_scripts/test_inaturalist.py

```diff
@@ -41,16 +23,125 @@
 # ---> Reading file photos.csv.gz
 # 36 records
 
+INAT = inaturalist.inaturalistDataIngester()
+RECORD0 = {
+    "foreign_id": 191018903,
+    "filetype": "jpg",
+    "license_url": "http://creativecommons.org/licenses/by-nc/4.0/",
+    "width": 818,
+    "height": 741,
+    "foreign_landing_url": "https://www.inaturalist.org/photos/191018903",
+    "image_url": "https://inaturalist-open-data.s3.amazonaws.com/photos/191018903/medium.jpg",
+    "creator": "tiwane",
+    "creator_url": "https://www.inaturalist.org/users/28",
+    "title": "Ranunculus occidentalis",
+    "tags": "Tracheophyta; Angiospermae; Magnoliopsida; Ranunculales; Ranunculaceae; Ranunculoideae; Ranunculeae; Ranunculus",
+}
+
+
+def test_get_next_query_params_no_prior():
+    expect = {"page_number": 0}
+    actual = INAT.get_next_query_params()
+    assert expect == actual
+
+
+def test_get_next_query_params_prior_0():
+    expect = {"page_number": 1}
+    actual = INAT.get_next_query_params({"page_number": 0})
+    assert expect == actual
+
+
 @pytest.mark.parametrize(
-    "file_name, expected",
+    "file_name, expect",
```

Comment:
Nit: almost all our other tests use `expected`, it'd be nice to have consistency here by changing `expect` and `expect0` to `expected`.

Reply:
Done!